test/upgrade/dynamic/service_postupgrade_test.go

```diff
@@ -0,0 +1,34 @@
+// +build postupgrade
+
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package upgrade
+
+import (
+	"knative.dev/serving/test/upgrade"
+	"testing"
+)
+
+func TestRunLatestServicePostUpgrade(t *testing.T) {
+	t.Parallel()
+	upgrade.UpdateService(upgrade.ServiceName, t, "v1beta1")
```

Comment:
Given the commonality between the this and the scale-to-zero tests below it might be a good candidate for sub-tests. It should reduce some boiler plate and duplication https://blog.golang.org/subtests

Reply:
Refactored test into subtests